### PR TITLE
bpo-38840: Fix test___all__ on platforms lacking

### PR DESCRIPTION
**Pull request by xdegaye**
a shared memory implementation

https://bugs.python.org/issue38840

### CODE REVIEW

File: Lib/multiprocessing/managers.py [hunk 1]
```diff
@@ -8,8 +8,7 @@
 # Licensed to PSF under a Contributor Agreement.
 #
 
-__all__ = [ 'BaseManager', 'SyncManager', 'BaseProxy', 'Token',
-            'SharedMemoryManager' ]
+__all__ = [ 'BaseManager', 'SyncManager', 'BaseProxy', 'Token']
```
Comment (corona10) [inline Lib/multiprocessing/managers.py]: nit
`['BaseManager', 'SyncManager', 'BaseProxy', 'Token']`
or 
`[ 'BaseManager', 'SyncManager', 'BaseProxy', 'Token' ]`
?

**Review by corona10 — APPROVED** [review]

File: Lib/multiprocessing/managers.py [hunk 2]
```diff
@@ -35,6 +34,7 @@
 try:
     from . import shared_memory
     HAS_SHMEM = True
+    __all__.append(SharedMemoryManager)
```
Comment (corona10) [inline Lib/multiprocessing/managers.py]:
```python
__all__.append('SharedMemoryManager')
``` 
?

**Review by corona10 — CHANGES_REQUESTED** [review]
I left some nit comments.

File: Lib/multiprocessing/managers.py (hunk 1 above)
Comment (xdegaye) [reply Lib/multiprocessing/managers.py]: Done in last commit.

File: Lib/multiprocessing/managers.py (hunk 2 above)
Comment (xdegaye) [reply Lib/multiprocessing/managers.py]: Done in last commit.

**Comment by xdegaye** [comment]
@corona10 
> I left some nit comments.
Thanks

**Pull request closed without merging**